saturn-r/postBuild

```diff
@@ -0,0 +1,37 @@
+# https://cran.rstudio.com/bin/linux/debian/
+sudo apt-key adv --keyserver keys.gnupg.net --recv-key 'E19F5F87128899B192B1A2C2AD5F960A256A04AF'
+
+sudo apt-get update
+sudo apt-get install -y \
+    --no-install-recommends \
+        software-properties-common
+
+sudo su -c \
+    "echo 'deb http://cloud.r-project.org/bin/linux/debian buster-cran40/' >>/etc/apt/sources.list"
+
+sudo apt update
+sudo apt install -y -t buster-cran40 \
+    r-base \
+    r-base-dev 
+
+sudo apt-get update
+sudo apt-get install -y \
+    jupyter-client \
+    devscripts \
+    libxml2 \
+    libxml2-dev \
+    libcurl4-openssl-dev \
+    libssl-dev \
+    libzmq3-dev \
+    qpdf
+
+# make it possible to `install.packages()` - write permissions required.
+sudo chown 1000:1000 -R /usr/local/lib/R/
+sudo chmod 777 -R /usr/local/lib/R/
+
+
+# install other packages
+sudo Rscript -e "install.packages(c('dplyr', 'ggplot2', 'caret', \
+ 'remotes', 'xgboost', 'lightgbm', 'data.table', \
+ 'Rcpp', 'reticulate', 'stringr', 'tidyr', 'IRkernel'), \
+ dependencies = c('LinkingTo', 'Depends', 'Imports'))"
```

Comment:
ok this one is very minor and feel free to ignore it, but could you make this one package per line, and alphabetical?

One package per line because then it's possible to use the git blame to easily figure out when a particular package was added. Alphabetical because the order of packages doesn't impact `install.packages()` in any meaningful way, but it does make it easier to visually scan the list to check if a particular package is currently in it.

```suggestion
sudo Rscript -e "install.packages(c( \
        'caret', \
        'data.table', \
        'dplyr', \
        'ggplot2', \
        'IRkernel', \
        'lightgbm', \
        'Rcpp', \
        'remotes', \
        'reticulate', \
        'stringr', \
        'tidyr', \
        'xgboost' \
    ), \
    dependencies = c('LinkingTo', 'Depends', 'Imports') \
    )"
```